content/en/blog/_posts/2023-11-20-volume-attributes-class/index.md

```diff
@@ -0,0 +1,162 @@
+# Kubernetes v1.29: VolumeAttributesClass for Volume Modification
```

Comment:
The blog post format needs to begin with these lines:

`---
layout: blog
title: "Kubernetes 1.29: VolumeAttributesClass for Volume Modification"
date: 2023-11-15
slug: kubernetes-1-29-volume-attributes-class
---

**Author:** Sunny Song (Google)`